phylo/src/likelihood/mod.rs

```diff
@@ -1,9 +1,9 @@
 use crate::substitution_models::FreqVector;
 
-pub trait LikelihoodCostFunction<'a, const N: usize> {
+pub trait LikelihoodCostFunction<'a> {
     type Model;
     type Info;
-    fn compute_log_likelihood(&self, model: &Self::Model) -> f64;
+    fn compute_log_likelihood(&self) -> f64;
     fn get_empirical_frequencies(&self) -> FreqVector;
```

Comment:
nit: similarly, `empirical_freq` would be enough

Reply:
Done.